config/locales/en.yml

```diff
@@ -865,9 +865,6 @@ en:
     label: Course Activity
     last_update: Last statistics update
     next_update: Next estimated update
-    update_statistics_link: See More
-    close_modal: Close
-    error_count_message: We encountered %{error_count} errors while updating your course.
```

Comment:
"The dashboard encounted X error(s) duing the most recent update."

Reply:
Done!